index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="container">
+      <div class="header">
+        <h1 id="title">Microverse Survey Form</h1>
+        <p id="description">Please take your time to help us improve our process</p>
+      </div>
+      <form id="survey-form">
+        <div id="name-div">
+          <label for="name" id="name-label">Name</label><br>
+          <input type="text" id="name" name="name" placeholder="enter your name" required>
+        </div>
+        <div id="email-div">
+          <label for="email" id="email-label">Email</label><br>
+          <input type="text" id="email" name="email" placeholder="enter your email" required>
+        </div>
+        <div>
+          <label for="number" id="number-label">Age</label><br>
+          <input type="number" id="number" name="age" placeholder="Age" step="1" min="18" max="40">
+        </div>
```

Comment:
I think using a `fieldset` tag is more appropriate and semantic than using a div tag. To know more about HTML semantic tags, you can read up on this [this](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and also [this] (https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure)